core/src/Revolution/modX.php

```diff
@@ -2647,12 +2647,12 @@ protected function _initCulture($options = null) {
         $this->cultureKey = $cultureKey;
         $this->setOption('cultureKey', $cultureKey);
 
-        if ($this->getOption('setlocale', $options, true)) {
```

Comment:
Why did you remove the setlocale if statement?

Reply:
ouch, I think that's by accident, let me fix that